bin/compile

```diff
@@ -93,22 +107,4 @@ echo " done"
 mkdir -p "$build/.profile.d"
 cat << EOF > "$build/.profile.d/boot-env.sh"
 export PATH="\$HOME/.bootbin:\$PATH"
-
-limit=\$(ulimit -u)
-case \$limit in
-256)   # 1X Dyno
-  default_java_opts="-Xmx384m -Xss512k"
-  ;;
-512)   # 2X Dyno
-  default_java_opts="-Xmx768m"
-  ;;
-32768) # PX Dyno
-  default_java_opts="-Xmx4g"
-  ;;
-*)
-  default_java_opts="-Xmx384m -Xss512k"
-  ;;
-esac
-
-export BOOT_JVM_OPTIONS=\${BOOT_JVM_OPTIONS:-"\${default_java_opts} -Dfile.encoding=UTF-8"}
```

Comment:
Yes, every Java process picks up JAVA_TOOL_OPTIONS
https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html

I think it's better keep this in jvm-common because it will be adjusted for things like new dyno types:
https://github.com/heroku/heroku-buildpack-jvm-common/blob/master/opt/jvmcommon.sh#L13-L14

But it does not hurt to keep it here too -- and other users might it better if this is more explicit.